AgentDiscoveries-Backend/src/main/resources/config.properties

```diff
@@ -1,6 +1,6 @@
 
 # Database properties
-database.url = jdbc:mysql://localhost:3306/agentdiscoveries
+database.url = jdbc:mysql://localhost:3306/agentdiscoveries?useSSL=false

```

Comment:
Lucky this isn't a super secure spy app :)

(Is this for local dev, or is it also required in production?)

Reply:
Local, will remove